routes/routes.movies.js

```diff
@@ -1,9 +1,40 @@
 const router = require("express").Router();
+const Movies = require("../models/movie.model");
+
+
 
 // all your routes here
 
+router.get("/movies", (req,res,next) =>{
+    Movies.find()
+        .then((moviesArr) => {
+            res.render("movies/movies", {movies: moviesArr})
+        })
+        .catch(err => console.log("error displaying movies on first route", err))
+})
+
+
+
+router.get("/movies/create", (req,res,next) => {
+    Movies.create()
+        .then((moviesArr) => {
+            res.render("movies/new-movie",{movies: moviesArr})
+        })
+        .catch(err => console.log("error while making a movie", err))
 
+})
```

Comment:
Same as for create celebrities route. 

You create the movie only once you receive information from the form (in POST route). However what you need is to get list of celebrities from the database , `Celebrity.find() ` , and you want to send that info to the view to display in dropdown